apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSink.java

```diff
@@ -123,6 +147,23 @@ public void updateLastSequenceNumber(long lastSequenceNumber) {
     }
   }
 
+  /**
+   * Writes out the transformation error to the database for the given message and given apiVersion.
+   *
+   * @param apiVersion The version of the api used to get the message.
+   * @param message The message that was being transformed when the error occurred.
+   * @param exception The exception that was thrown while transforming the message.
+   * @throws IOException If there was an issue writing to the database.
+   */
+  @Override
+  public void writeError(
+      String apiVersion, TMessage message, DataTransformer.TransformationException exception)
+      throws IOException {
+    entityManager.getTransaction().begin();
```

Comment:
Is there already a transaction block further up in the stack?  I know we're writing records to the DB in batches, so I thought that maybe there was a transaction block at that level, in which case I think this would cause issues starting a new transaction while were already in one.